### PR TITLE
:tractor: relocated ~/.moban/repos to appdirs/user_cache_dir()/repos,…

### PR DESCRIPTION
**Pull request by chfw**
delivers #173

### CODE REVIEW

File: .moban.cd/moban.yml [hunk 1]
```diff
@@ -15,9 +15,10 @@ keywords:
   - jinja2
   - moban
 dependencies:
-  - ruamel.yaml
+  - ruamel.yaml==0.15.86
```
Comment (jayvdb) [inline .moban.cd/moban.yml]: was there some feature which needed .86 ?

https://build.opensuse.org/package/show/devel:languages:python/python-ruamel.yaml is on .80

I'll try updating it, but that will take longer.

File: .moban.cd/moban.yml [hunk 2]
```diff
@@ -15,9 +15,10 @@ keywords:
...
   - jinja2>=2.7.1
-  - lml>=0.0.7
+  - lml>=0.0.9
+  - appdirs==1.4.3
```
Comment (jayvdb) [inline .moban.cd/moban.yml]: We can probably allow lower, but this is OK with most updated distros, as 1.4.3 is quite old.

e.g. https://build.opensuse.org/package/show/devel:languages:python/python-appdirs

File: .moban.cd/moban.yml (hunk 1 above)
Comment (chfw) [reply .moban.cd/moban.yml]: just wanted to nominate a version instead of any.

Comment (chfw) [reply .moban.cd/moban.yml]: what's the one available on OpenSuse? without considering distros, any working version would be OK.

File: .moban.cd/moban.yml (hunk 2 above)
Comment (chfw) [reply .moban.cd/moban.yml]: would it be too much picky when it comes to OpenSuse?

File: .moban.cd/moban.yml (hunk 1 above)
Comment (jayvdb) [reply .moban.cd/moban.yml]: build.opensuse.org/package/show/devel:languages:python/python-ruamel.yaml is on .80 ; that is good enough for now.

**Pull request merged**